pkg/appcontext/context.go

```diff
@@ -23,11 +25,23 @@ func WithLogger(ctx context.Context, logger *zap.Logger) context.Context {
 }
 
 // Logger returns the context's logger
+// DEPRECATED - prefer ZLogger going forward
 func Logger(ctx context.Context) (*zap.Logger, bool) {
 	logger, ok := ctx.Value(loggerKey).(*zap.Logger)
 	return logger, ok
 }
 
+// ZLogger will always return something that functions
+// as a zap.Logger, even if it wasn't already placed
+// on the context
+func ZLogger(ctx context.Context) *zap.Logger {
```

Comment:
I also considered `ZapLogger`, as this functionality is forever associated with the `zap` package... But opted for shorter as initial effort... Open to changing if folks have strong feelings about it.